test/tools/jsontests/StateTests.cpp

```diff
@@ -189,7 +189,6 @@ BOOST_AUTO_TEST_CASE(stBadOpcode){}
 
 //New Tests
 BOOST_AUTO_TEST_CASE(stArgsZeroOneBalance){}
-BOOST_AUTO_TEST_CASE(stEWASMTests){}
```

Comment:
Does that mean we can't use testeth anymore with https://github.com/ewasm/hera ?

Reply:
you can use your fork sure.
The issue is that new forks will not be supported in testeth.

Reply:
What fork do you mean?

Reply:
I mean if you have stEWASMTests  somewhere and use testeth to run it.